torch/_higher_order_ops/print.py

```diff
@@ -25,6 +33,38 @@ def __call__(self, format_str: str, **kwargs: object) -> object:
 print = Print()
 
 
+def trace_print(proxy_mode, func_overload, format_str, **kwargs):
+    def _unwrap_proxy(e):
+        if not isinstance(e, (torch.Tensor, torch.SymInt, torch.SymFloat)):
```

Comment:
can you add a test which prints tensors and symints?

Reply:
Added print tensors, symints and float. Thanks!